.github/CONTRIBUTING.md

```diff
@@ -20,50 +21,87 @@ This document is a set of guidelines for contributors, nothing is set in stone s
 
 ## How to contribute
 
-There are plenty of ways to contribute, not only by submitting new code! Reporting bugs, adding/improving documentation and tests is just as important to us.
+There are plenty of ways to contribute, not only by submitting new code!
+Reporting bugs, adding/improving documentation and tests is just as important
+to us.
+
 
 ### Reporting bugs
 
-* First, please make sure the bug was not already reported by searching on GitHub under [issues](https://github.com/h2oai/datatable/issues).
-* Only when you are sure the bug has not yet been reported, [open a new issue](https://github.com/h2oai/datatable/issues/new).
-* Please follow the [issue template](ISSUE_TEMPLATE.md) and provide as many details as possible. A clear, but concise, title, your environment, and a [MCVE](https://stackoverflow.com/help/mcve) will make everyone's life easier.
+* First, please make sure the bug was not already reported by searching on
+  GitHub under [issues](https://github.com/h2oai/datatable/issues).
+
+* Only when you are sure the bug has not yet been reported,
+  [open a new issue](https://github.com/h2oai/datatable/issues/new).
+* Please follow the [issue template](ISSUE_TEMPLATE.md) and provide as many
+  details as possible. A clear, but concise, title, your environment, and a
+  [MCVE](https://stackoverflow.com/help/mcve) will make everyone's life easier.
+
 
 ### Enhancements and Feature requests
 
 We are open to new enhancements/feature requests!
 
-* Just like with bugs, please check whether there is already an existing issue in the tracker.
-* If not, then you can reach create a new issue with an appropriate label (`feature request` or `enhancement`).
+* Just like with bugs, please check whether there is already an existing issue
+  in the tracker.
+
+* If not, then you can reach create a new issue with an appropriate label
```

Comment:
Shouldn't it be just `then you can create`?

Reply:
👍